lit_parrot/adapter.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, config: Config, block_idx: int) -> None:
             self.adapter_wte = nn.Embedding(config.adapter_prompt_length, config.n_embd)
             # gate for adaption
             self.gating_factor = torch.nn.Parameter(torch.zeros(1))
+            self.adapter_cache_kv = []
```

Comment:
Where is this used? The adapter cache kv is saved in Parrot

```suggestion
```

Reply:
thanks for catching this, I removed this line.